### PR TITLE
add strings.xml for French

### PR DESCRIPTION
**Pull request by perepepepa**

### CODE REVIEW

**Comment by dessalines** [comment]
Lots of compile errors.

**Comment by perepepepa** [comment]
Hi I fixed the compile errors and also some incorrect translations.
The errors were some unescaded apostrophies.
Now the pull request is stuck with woodpecker, it will unstuck by himself or I have to do something ?

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="login_instance_placeholder">ex.: lemmy.ml</string>
+    <string name="login_email_or_username">Courriel ou Nom Utilisateur</string>
+    <string name="login_login">Se connecter</string>
+    <string name="login_back">Précédent</string>
+    <string name="person_profile_joined">Profil créé</string>
+    <string name="person_profile_posts">%1$s publication(s)</string>
+    <string name="person_profile_comments">%1$s commentaires</string>
+    <string name="person_profile_back">Précédent</string>
+    <string name="person_profile_block_person">Bloquer utilisateur</string>
+    <string name="person_profile_report_person">Signaler utilisateur</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:96]:
```suggestion
    <string name="person_profile_report_person">Signaler l'utilisateur</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="comment_node_view_post">Afficher la publication</string>
+    <string name="comment_node_view_context">Afficher le contexte</string>
+    <string name="comment_body_removed">*Supprimé*</string>
+    <string name="comment_body_deleted">*Effacé*</string>
+    <string name="comment_edit_edit">Editer</string>
+    <string name="comment_edit_type_your_comment">Tapez votre commentaire</string>
+    <string name="comment_edit_back">Précédent</string>
+    <string name="comment_mention_node_view_source">Afficher Markdown</string>
+    <string name="comment_mention_node_copy_permalink">Copier permalink</string>
+    <string name="comment_mention_node_report_comment">Signaler commentaire</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:20]:
```suggestion
    <string name="comment_mention_node_report_comment">Signaler le commentaire</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="pictrs_image_upload_image">Charger une image</string>
+    <string name="time_ago_ago">%1$s il y a %2$s</string>
+    <string name="community_list_search">Chercher…</string>
+    <string name="community_users_month">%1$s utilisateur(s)/mois</string>
+    <string name="community_joined">Suivie</string>
+    <string name="community_subscribe">Suivre</string>
+    <string name="community_pending">En cours</string>
+    <string name="community_back">Précédent</string>
+    <string name="community_refresh">Rafraîchir</string>
+    <string name="community_community_info">Info communauté</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:65]:
```suggestion
    <string name="community_community_info">Infos sur la communauté</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="community_block_community">Bloquer communauté</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:66]:
```suggestion
    <string name="community_block_community">Bloquer la communauté</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="community_link_users_month">%1$s utilisateurs/mois</string>
+    <string name="home_subscribed">Communautés suivies</string>
+    <string name="home_local">Seule instance locale</string>
+    <string name="home_all">Toutes les communautés</string>
+    <string name="home_saved">Publications et commentaires enregistrés</string>
+    <string name="home_profile">Profil</string>
+    <string name="home_inbox">Boîte De Réception</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:73]:
```suggestion
    <string name="home_inbox">Boîte de réception</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="home_settings">Réglages</string>
+    <string name="home_subscriptions">Communautés Suivies</string>
+    <string name="home_add_account">Ajouter Un Profil</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:76]:
```suggestion
    <string name="home_add_account">Ajouter un compte</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 7]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="home_switch_to">Passer à %1$s/%2$s</string>
+    <string name="home_sign_out">Quitter</string>
+    <string name="home_menu">Menu</string>
+    <string name="home_refresh">Rafraîchir</string>
+    <string name="home_post_view_mode">Mode d\'affichage</string>
+    <string name="home_site_info">Informations sur l\'instance</string>
+    <string name="inbox_back">Précédent</string>
+    <string name="inbox_inbox">Boîte De Réception</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:84]:
```suggestion
    <string name="inbox_inbox">Boîte de réception</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 8]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="login_password">Mot de passe</string>
+    <string name="login_instance">Instance</string>
+    <string name="login_instance_placeholder">ex.: lemmy.ml</string>
+    <string name="login_email_or_username">Courriel ou Nom Utilisateur</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:88]:
```suggestion
    <string name="login_email_or_username">Courriel ou nom d'utilisateur</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 9]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="login_login">Se connecter</string>
+    <string name="login_back">Précédent</string>
+    <string name="person_profile_joined">Profil créé</string>
+    <string name="person_profile_posts">%1$s publication(s)</string>
+    <string name="person_profile_comments">%1$s commentaires</string>
+    <string name="person_profile_back">Précédent</string>
+    <string name="person_profile_block_person">Bloquer utilisateur</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:95]:
```suggestion
    <string name="person_profile_block_person">Bloquer l'utilisateur</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 10]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="create_post_close">Quitter</string>
+    <string name="create_post_copy_suggested_title">Copier titre suggéré : %1$s</string>
+    <string name="create_post_community">Communauté</string>
+    <string name="create_post_a_title_here">un titre…</string>
+    <string name="post_edit_edit_post">Editer la publication</string>
+    <string name="post_edit_close">Quitter</string>
+    <string name="post_edit_body_placeholder">Contenuto</string>
+    <string name="post_activity_comments">Commentaires</string>
+    <string name="post_listing_comments">%1$s commentaires</string>
+    <string name="post_listing_new">( %1$s nuovi )</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:108]:
```suggestion
    <string name="post_listing_new">( %1$s nouveau )</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 11]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="person_profile_report_person">Signaler utilisateur</string>
+    <string name="person_profile_activity_moderates">Modérateur</string>
+    <string name="create_post_create_post">Créer une publication</string>
+    <string name="create_post_close">Quitter</string>
+    <string name="create_post_copy_suggested_title">Copier titre suggéré : %1$s</string>
+    <string name="create_post_community">Communauté</string>
+    <string name="create_post_a_title_here">un titre…</string>
+    <string name="post_edit_edit_post">Editer la publication</string>
+    <string name="post_edit_close">Quitter</string>
+    <string name="post_edit_body_placeholder">Contenuto</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:105]:
```suggestion
    <string name="post_edit_body_placeholder">Contenu</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 12]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="person_profile_comments">%1$s commentaires</string>
+    <string name="person_profile_back">Précédent</string>
+    <string name="person_profile_block_person">Bloquer utilisateur</string>
+    <string name="person_profile_report_person">Signaler utilisateur</string>
+    <string name="person_profile_activity_moderates">Modérateur</string>
+    <string name="create_post_create_post">Créer une publication</string>
+    <string name="create_post_close">Quitter</string>
+    <string name="create_post_copy_suggested_title">Copier titre suggéré : %1$s</string>
+    <string name="create_post_community">Communauté</string>
+    <string name="create_post_a_title_here">un titre…</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:102]:
```suggestion
    <string name="create_post_a_title_here">Un titre…</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 13]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="post_edit_edit_post">Editer la publication</string>
+    <string name="post_edit_close">Quitter</string>
+    <string name="post_edit_body_placeholder">Contenuto</string>
+    <string name="post_activity_comments">Commentaires</string>
+    <string name="post_listing_comments">%1$s commentaires</string>
+    <string name="post_listing_new">( %1$s nuovi )</string>
+    <string name="post_listing_comments_count">%1$s commentaires</string>
+    <string name="post_listing_go_to">Aller vers %1$s</string>
+    <string name="post_listing_copy_link">Copier lien</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:111]:
```suggestion
    <string name="post_listing_copy_link">Copier le lien</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 14]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="post_listing_copy_permalink">Copier permalink</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:112]:
```suggestion
    <string name="post_listing_copy_permalink">Copier le permalink</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 15]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="post_listing_restore">Restaurer</string>
+    <string name="post_listing_delete">Supprimer</string>
+    <string name="private_message_to">"à "</string>
+    <string name="private_message_from">"de "</string>
+    <string name="private_message_reply_reply">Répondre</string>
+    <string name="private_message_reply_back">Précédent</string>
+    <string name="private_message_reply_type_your_message_placeholder">Saisir le message</string>
+    <string name="create_report_report">Signaler</string>
+    <string name="create_report_back">Précédent</string>
+    <string name="create_report_type_your_reason">Exprimer sa motivation</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:125]:
```suggestion
    <string name="create_report_type_your_reason">Ecrivez la raison</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 16]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="settings_about_developer_matrix_chatroom">Chat Matrix des développeurs</string>
+    <string name="settings_about_donate_to_jerboa_development">Soutenir le développement de Jerboa</string>
+    <string name="settings_about_join_c_jerboa">Rejoindre c/jerboa</string>
+    <string name="settings_about_follow_on_mastodon">Suivez-nous sur Mastodon</string>
+    <string name="settings_about_open_source">Open Source</string>
+    <string name="settings_about_source_code">Code Source</string>
+    <string name="settings_about_source_code_subtitle_part1">"Jerboa est un logiciel libre et gratuit, publié sous une licence "</string>
+    <string name="settings_about_source_code_subtitle_part2"> GNU Affero General Public License v3.0</string>
+    <string name="account_settings_remove_current_avatar">Supprimer l\'avatar actuel</string>
+    <string name="account_settings_display_name">Nom affiché</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:140]:
```suggestion
    <string name="account_settings_display_name">Nom d'affichage</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 17]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="AppBars_posts">%1$s publication(s)</string>
+    <string name="AppBars_comments">%1$s commentaire(s)</string>
+    <string name="dialogs_top_day">Meilleures du jour</string>
+    <string name="dialogs_top_week">Meilleures de la semaine</string>
+    <string name="dialogs_top_month">Meilleures du mois</string>
+    <string name="dialogs_top_year">Meilleures de l\'année</string>
+    <string name="dialogs_top_all_time">Meilleures de tous les temps</string>
+    <string name="dialogs_subscribed">Communautés suivies</string>
+    <string name="dialogs_local">Seule instance locale</string>
+    <string name="dialogs_all">Tout</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:42]:
```suggestion
    <string name="dialogs_all">Tous</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 18]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="account_settings_local">Locals</string>
+    <string name="account_settings_subscribed">Suivie</string>
+    <string name="account_settings_default_listing_type">Classement par défaut</string>
+    <string name="account_settings_active">Actifs</string>
+    <string name="account_settings_hot">Populaires</string>
+    <string name="account_settings_new">Nouveaux</string>
+    <string name="account_settings_topday">Meilleures du jour</string>
+    <string name="account_settings_topweek">Meilleures de la semaine</string>
+    <string name="account_settings_topmonth">Meilleures du mois</string>
+    <string name="account_settings_topyear">Meilleures de l\'Année</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:156]:
```suggestion
    <string name="account_settings_topyear">Meilleures de l\'année</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 19]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="account_settings_topall">Meilleures de tous les temps</string>
+    <string name="account_settings_mostcomments">Plus de commentaires</string>
+    <string name="account_settings_newcomments">Nouveaux commentaires</string>
+    <string name="account_settings_default_sort_type">Ordre par defaut</string>
+    <string name="account_settings_show_nsfw">Afficher NSFW</string>
+    <string name="account_settings_show_avatars">Afficher avatar</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:162]:
```suggestion
    <string name="account_settings_show_avatars">Afficher les avatars</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 20]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="account_settings_new">Nouveaux</string>
+    <string name="account_settings_topday">Meilleures du jour</string>
+    <string name="account_settings_topweek">Meilleures de la semaine</string>
+    <string name="account_settings_topmonth">Meilleures du mois</string>
+    <string name="account_settings_topyear">Meilleures de l\'Année</string>
+    <string name="account_settings_topall">Meilleures de tous les temps</string>
+    <string name="account_settings_mostcomments">Plus de commentaires</string>
+    <string name="account_settings_newcomments">Nouveaux commentaires</string>
+    <string name="account_settings_default_sort_type">Ordre par defaut</string>
+    <string name="account_settings_show_nsfw">Afficher NSFW</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:161]:
```suggestion
    <string name="account_settings_show_nsfw">Afficher les publications NSFW</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 21]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="account_settings_show_avatars">Afficher avatar</string>
+    <string name="account_settings_show_read_posts">Afficher les publications déjà lues</string>
+    <string name="account_settings_bot_account">Compte bot</string>
+    <string name="account_settings_show_bot_accounts">Afficher compte bot</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:165]:
```suggestion
    <string name="account_settings_show_bot_accounts">Afficher les comptes de robots</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 22]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="account_settings_topmonth">Meilleures du mois</string>
+    <string name="account_settings_topyear">Meilleures de l\'Année</string>
+    <string name="account_settings_topall">Meilleures de tous les temps</string>
+    <string name="account_settings_mostcomments">Plus de commentaires</string>
+    <string name="account_settings_newcomments">Nouveaux commentaires</string>
+    <string name="account_settings_default_sort_type">Ordre par defaut</string>
+    <string name="account_settings_show_nsfw">Afficher NSFW</string>
+    <string name="account_settings_show_avatars">Afficher avatar</string>
+    <string name="account_settings_show_read_posts">Afficher les publications déjà lues</string>
+    <string name="account_settings_bot_account">Compte bot</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:164]:
```suggestion
    <string name="account_settings_bot_account">Compte de robot</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 23]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="account_settings_show_bot_accounts">Afficher compte bot</string>
+    <string name="account_settings_show_scores">Afficher les scores</string>
+    <string name="account_settings_show_notifications_for_new_posts">Envoyer des notifications pour les nouvelles publications</string>
+    <string name="account_settings_send_notifications_to_email">Envoyer des notifications par courrier électronique</string>
+    <string name="account_settings_save_settings">Enregistrer les réglages</string>
+    <string name="account_settings_activity_account_settings">Réglages du profil</string>
+    <string name="look_and_feel_look_and_feel">Apparence</string>
+    <string name="look_and_feel_font_size">Taille De La Police : %1$s</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:172]:
```suggestion
    <string name="look_and_feel_font_size">Taille de la police : %1$s</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 24]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="look_and_feel_theme">Thème</string>
+    <string name="look_and_feel_theme_system">Du système</string>
+    <string name="look_and_feel_theme_system_black">Du système noir</string>
+    <string name="look_and_feel_theme_light">Clair</string>
+    <string name="look_and_feel_theme_dark">Foncé</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:177]:
```suggestion
    <string name="look_and_feel_theme_dark">Sombre</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 25]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="look_and_feel_theme_black">Noir</string>
+    <string name="look_and_feel_theme_color">Couleur du thème</string>
+    <string name="look_and_feel_post_view">Mode d\'affichage</string>
+    <string name="look_and_feel_post_view_card">Carte</string>
+    <string name="look_and_feel_post_view_small_card">Carte petite</string>
+    <string name="look_and_feel_post_view_list">Liste</string>
+    <string name="look_and_feel_show_navigation_bar">Afficher la barre de navigation </string>
+    <string name="look_and_feel_use_custom_tabs">Utiliser des onglets personnalisés</string>
+    <string name="settings_activity_settings">Réglages</string>
+    <string name="settings_activity_look_and_feel">Apparence interface</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:187]:
```suggestion
    <string name="settings_activity_look_and_feel">Apparence de l'interface</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 26]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="community_blocked">%1$s bloquée</string>
+    <string name="comment_mention_node_permalink_copied">Permalink copié</string>
+    <string name="person_routines_blocked">%1$s bloquée</string>
+    <string name="utils_login_first">Se connecter d\'abord</string>
+    <string name="login_view_model_incorrect_login">Nom d\'utilisateur ou mot de passe incorrect</string>
+    <string name="login_view_model_is_not_a_lemmy_instance">%1$s n\'est pas une instance de Lemmy</string>
+    <string name="create_report_view_model_report_created">Rapport créé</string>
+    <string name="about_social">Social</string>
+    <string name="app_bars_dot_spacer">·</string>
+    <string name="comment_node_in">"in "</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:199]:
```suggestion
    <string name="comment_node_in">"dans "</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 27]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="look_and_feel_activity_show_content_for_collapsed_comments">Afficher contenuto per commentaires minimizzati</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:200]:
```suggestion
    <string name="look_and_feel_activity_show_content_for_collapsed_comments">Afficher le contenu des commentaires minimisés</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 28]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="comment_node_permalink_copied">Permalink copié</string>
+    <string name="comment_reply_node_permalink_copied">Permalink copié</string>
+    <string name="post_listing_link_copied">Lien copié</string>
+    <string name="post_listing_permalink_copied">Permalink copié</string>
+    <string name="comment_node_block">Bloquer %1$s</string>
+    <string name="comment_reply_node_copy_permalink">Copier Permalink</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:206]:
```suggestion
    <string name="comment_reply_node_copy_permalink">Copier le permalink</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 29]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="person_profile_activity_about">Info</string>
+    <string name="person_profile_activity_posts">Publication</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:208]:
```suggestion
    <string name="person_profile_activity_posts">Publications</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 30]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="comment_mention_node_go_to">Aller sur</string>
+    <string name="comment_reply_node_go_to">Aller sur</string>
+    <string name="account_settings_old">Anciennes</string>
+    <string name="community_list_back">Précédent</string>
+    <string name="moreOptions">Afficher plus d\'options</string>
+    <string name="lessOptions">Afficher moins d\'options</string>
+    <string name="bottomBar_home">Accueil</string>
+    <string name="bottomBar_bookmarks">Favoris</string>
+    <string name="bottomBar_profile">Mon profil</string>
+    <string name="bottomBar_inbox">Boîte De Réception</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:231]:
```suggestion
    <string name="bottomBar_inbox">Boîte de réception</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 31]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="commentFooter_reply">Répondre à ce commentaire</string>
+    <string name="dialog_moreOptions">Afficher plus d\'options</string>
+    <string name="pickImage_imagePreview">Aperçu de l\'image chargée</string>
+    <string name="commentEdit_save">Enregistrer les modifications au commentaire</string>
+    <string name="commentReply_send">Envoyer la réponse</string>
+    <string name="community_sortBy">Trier par</string>
+    <string name="floating_createPost">Créer publication</string>
+    <string name="community_icon">Symbole de la communauté</string>
+    <string name="form_submit">Envoyer</string>
+    <string name="createPost_selectCommunity">Selectionner une communauté de la liste</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:244]:
```suggestion
    <string name="createPost_selectCommunity">Selectionner une communauté dans liste</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 32]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="markUnread">Marquer comme non lu</string>
+    <string name="comment_save">Enregistrer le commentaire</string>
+    <string name="comment_unsave">Rejeter le commentaire</string>
+    <string name="commentReply_link">Aller vers la réponse</string>
+    <string name="homeHeader_filter">Selection du filtre</string>
+    <string name="selectSort">Selectionner trie</string>
+    <string name="inbox_filter">Selection du filtre pour la boîte de réception</string>
+    <string name="inbox_markAllRead">Marquer tout comme déjà lu</string>
+    <string name="markdownHelper_preview">Afficher aperçu</string>
+    <string name="markdownHelper_insertLink">Inserer lien</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:256]:
```suggestion
    <string name="markdownHelper_insertLink">Inserer un lien</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 33]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="markdownHelper_insertImage">Inserer image</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:257]:
```suggestion
    <string name="markdownHelper_insertImage">Inserer une image</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 34]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="markdownHelper_formatBold">Format en gras</string>
+    <string name="markdownHelper_formatItalic">Format en italique</string>
+    <string name="markdownHelper_insertQuote">Inserer citation</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:260]:
```suggestion
    <string name="markdownHelper_insertQuote">Inserer une citation</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 35]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="markdownHelper_insertList">Inserer liste</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:261]:
```suggestion
    <string name="markdownHelper_insertList">Inserer une liste</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 36]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="markdownHelper_insertHeader">Inserer titre</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:262]:
```suggestion
    <string name="markdownHelper_insertHeader">Inserer un titre</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 37]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="personProfile_viewAvatar">Afficher avatar</string>
+    <string name="personProfile_viewBanner">Afficher banner</string>
+    <string name="postListing_goToCommunity">Aller vers la communauté</string>
+    <string name="postListing_featuredLocal">En vedette localement</string>
+    <string name="postListing_featuredCommunity">En vedette dans la communauté</string>
+    <string name="postListing_locked">Verrouillé</string>
+    <string name="postListing_deleted">Supprimé</string>
+    <string name="postListing_reply">Répondre au message</string>
+    <string name="privateMessage_reply">Répondre au message</string>
+    <string name="voteToggle">Changer de Vote</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:279]:
```suggestion
    <string name="voteToggle">Changer de vote</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 38]
```diff
@@ -0,0 +1,292 @@
...
+    <string name="home_communities">Liste des communautés suivies</string>
+    <string name="bottomBar_label_bookmarks">Enregistrés</string>
+    <string name="bottomBar_label_home">Accueil</string>
+    <string name="bottomBar_label_inbox">Réception</string>
+    <string name="bottomBar_label_profile">Profil</string>
+    <string name="bottomBar_label_search">Recherche</string>
+    <string name="bottomBar_search">Aller à la recherche</string>
+    <string name="dialogs_old">Anciennes</string>
+    <string name="look_and_feel_show_action_bar_for_comments">Montrer par default la barre d\'action des commentaires</string>
+    <string name="look_and_feel_show_voting_arrows_list_view">Montrer le fleche de votation dans la liste</string>
```
Comment (Krafting) [inline app/src/main/res/values-fr/strings.xml:290]:
```suggestion
    <string name="look_and_feel_show_voting_arrows_list_view">Montrer la fleche de vote dans la liste</string>
```

**Review by Krafting — CHANGES_REQUESTED** [review]
Few tweaks

**Pull request merged**